packages/trace-viewer/src/ui/attachmentsTab.tsx

```diff
@@ -139,3 +139,10 @@ function attachmentURL(attachment: Attachment) {
     return 'sha1/' + attachment.sha1 + '?trace=' + encodeURIComponent(attachment.traceUrl);
   return 'file?path=' + encodeURIComponent(attachment.path!);
 }
+
+function downloadURL(attachment: Attachment) {
+  const url = attachmentURL(attachment) + '&dn=' + encodeURIComponent(attachment.name);
```

Comment:
I'd love to see `attachmentURL` using `URLSearchParams` inside and accept a `Record<string, string>` of additional options.